docs/src/modules/components/MaterialUIExampleCollection.js

```diff
@@ -87,6 +87,12 @@ const examples = [
     link: 'https://github.com/mui/material-ui/tree/master/examples/material-ui-gatsby',
     src: '/static/images/examples/gatsby.svg',
   },
+  {
+    name: 'React-admin',
+    label: 'View TS example',
+    link: 'https://github.com/mui/material-ui/tree/master/examples/material-ui-react-admin',
```

Comment:
Can we move the source to another location? I doubt we should maintain this here.

Reply:
Done!

Reply:
I think that we could rework the whole section.

1. "Example projects" as a header doesn't make much sense to me. Instead, it could be "Framework guides", like in https://tailwindcss.com/docs/installation/framework-guides, https://www.tremor.so/docs/getting-started/installation, https://nextui.org/docs/frameworks/nextjs, https://ui.shadcn.com/docs/installation, https://mantine.dev/getting-started/#framework-guide. Some call it "Integration", e.g. https://reshaped.so/content/docs/getting-started/integrations/tailwind but this a very broad term. I think that in cases like using it with Storybook, [icon libraries](https://mui.com/joy-ui/integrations/icon-libraries/), etc. it's doesn't need to be part of the getting started area of the docs, these concerns only come later.
2. Have a dedicated page for each framework, we started with https://mui.com/joy-ui/integrations/next-js-app-router/, I would scale this to everything. We need both an example and a step-by-step guide on how to create the example. Today, I don't think that we play the step-by-step guide card enough but it's important, when you already have a product on a given framework, you don't want to reverse engineer examples to figure out what to add. This is also true when you are just getting started, and need to rely on two integrations, e.g. Next.js + Tailwind CSS.

Reply:
Issue created #40062